app/helpers/filters_helper.rb

```diff
@@ -45,12 +45,12 @@ def selected_option(filter, filter_type)
   end
 
   def owning_organisations_filter_options(user)
```

Comment:
Could we update/add tests for these helper methods?

Reply:
good idea, have done :)